frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick-start-devperspective.feature

```diff
@@ -95,26 +86,42 @@ Feature: Quick Starts card in developer console
               And user selects Yes in Check your Work section
               And user clicks on Next button in Step 5
               And user selects Yes in Check your Work section
-              And user clicks on Restart
-             Then user can see Start, Close and restart button
+              And user clicks on Next
+             Then user can see Start, Close and Restart button
               And user can see link to "Add health checks to your sample application" quick start
               And user can see 5 Steps visible with checkmark in Green color
 
 
         @odc-5010 @to-do
         Scenario: Restart action on Quick Start card: QS-03-TC09
-            Given user is at Quick Start card page
+            Given user is at Quick Start catalog page
              When user clicks "Get started with a sample application" Quick Start
-              And user starts the tour
-              And user completes the tour
+              And user Start button
+              And user completes the Quick Start
               And user clicks on Restart
              Then user can see Start button
-              And user can see 5 Steps visible for the tour
+              And user can see 5 Steps visible for the Quick Start
 
 
         @odc-5010 @manual
         Scenario: Resizing Quick Start drawer: QS-03-TC10
-            Given user is at Quick Start card page
+            Given user is at Quick Start catalog page
              When user clicks "Get started with a sample application" Quick Start
               And user drags the Quick Start drawer left and right from the left side of the panel
              Then user is able to resize the Quick Start drawer
+
+
+        @odc-5010 @manual
+        Scenario: Disabling Quick Start: QS-03-TC11
+            Given user is logged in as an admin
+             When user searches console in Resources dropdown
+              And user selects Console with apiVersion operator.openshift.io/v1 option from Resources dropdown
+              And user clicks on "cluster" Name in Consoles
+              And user switches to YAML tab
+              And user adds spec.customization.quickStarts.disabled section
+              And user inserts Add project access roles Snippet from the Sidebar under spec.customization.projectAccess section
```

Comment:
why do you need project access for quick starts

Reply:
Removed